test_scripts/Polices/build_options/P_ATF_Register_NewApp_not_exist_inLocalPT_start_PTU_for_NewApp_PROPRIETARY.lua

```diff
@@ -61,29 +62,69 @@ local registerAppInterfaceParams =
 commonSteps:DeleteLogsFileAndPolicyTable()
 
 --[[ General Settings for configuration ]]
-Test = require('connecttest')
+Test = require('user_modules/connecttest_resumption')
+
+function Test:Precondition_connectMobile()
+  self:connectMobile()
+end
+
+function Test:Precondition_StartSession()
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession:StartService(7)
+end
 
 --[[ Preconditions ]]
 commonFunctions:newTestCasesGroup ("Preconditions")
-function Test:Precondition_PolicyUpdateStarted()
+function Test:Precondition_RegisterApp_trigger()
+  local CorIdRegister = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.registerAppInterfaceParams.appName }})
+  EXPECT_RESPONSE(CorIdRegister, { success = true, resultCode = "SUCCESS" })
+  EXPECT_NOTIFICATION("OnHMIStatus", { systemContext = "MAIN", hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE"})
 
   EXPECT_HMICALL ("BasicCommunication.PolicyUpdate")
   :Do(function(_,data)
       self.hmiConnection:SendResponse(data.id, "BasicCommunication.PolicyUpdate", "SUCCESS", {})
     end)
+
+  EXPECT_NOTIFICATION("OnSystemRequest")--, {requestType = "LOCK_SCREEN_ICON_URL"} )
+  :ValidIf(function(_,data)
+      if(data.payload.requestType ~= "LOCK_SCREEN_ICON_URL") then
+        commonFunctions:printError("requestType should be PROPRIETARY")
+        return false
+      end
+      return true
+    end)
+end
+
+function Test:Precondition_PolicyUpdateStarted()
+  local policy_file_path = commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath") .. "/"
+
   local RequestIdGetURLS = self.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
-  EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {url = "http://policies.telematics.ford.com/api/policies"}}})
+  EXPECT_HMIRESPONSE(RequestIdGetURLS,{result = {code = 0, method = "SDL.GetURLS", urls = {{url = "http://policies.telematics.ford.com/api/policies"}}}})
   :Do(function(_,_)
       self.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
         {
           requestType = "PROPRIETARY",
           url = "http://policies.telematics.ford.com/api/policies",
           appID = self.applications ["Test Application"],
-          fileName = "sdl_snapshot.json"
+          fileName = policy_file_path .. "sdl_snapshot.json"
         }
       )
     end)
-  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY" })
+  EXPECT_NOTIFICATION("OnSystemRequest")
+  :Do(function(_, data)
+      if not (data.binaryData ~= nil and string.len(data.binaryData) > 0) then
+        self:FailTestCase("PTS was not sent to Mobile in payload of OnSystemRequest")
+      end
+    end)
+  :ValidIf(function(_,data)
```

Comment:
@dboltovskyi, yes, because of defect "[GENIVI] SDL is using outdated Mobile_API.xml"
For unclear reason this issue is observed for now only in this script.